src/bin/pgcopydb/dump_restore.c

```diff
@@ -804,21 +810,20 @@ copydb_write_restore_list(CopyDataSpec *specs, PostgresDumpSection section)
 	}
 
 	FILE *listFile = fopen_with_umask(listFilename, "wb", FOPEN_FLAGS_W, 0644);
-	if (!listFilename)
+	if (listFile == NULL)
 	{
 		log_error("Failed to open list file \"%s\": %m", listFilename);
 		return false;
 	}
-	ArchiveContext context = { 0 };
-	context.specs = specs;
-	context.line = line;
-	context.listFile = listFile;
+
+	ArchiveContext context = { .specs = specs, .line = line, .listFile = listFile };
 
 	bool res = archive_iter(listOutFilename, &context,
-							copydb_write_restore_list_callback);
+							copydb_write_restore_list_hook);
 	if (!res)
 	{
-		log_error("Failed to write filtered pg_restore list file at \"%s\"",
+		log_error("Failed to write filtered pg_restore list file at \"%s\""
+				  ", see above for details",
```

Comment:
We have 210 occurrences of "see above for details" with the coma on the previous line, why change the code style in this specific part?